Zappy_IA_Report/DTC/Code/game.py

```
@@ -0,0 +1,45 @@
+import process
+import protocole
+import socket
+import threading
+
+threads = []
+clients = []
+
+def add_client(client) -> None :
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)